apps/src/applab/Visualization.jsx

```diff
@@ -0,0 +1,32 @@
+var applabConstants = require('./constants');
+
+var styles = {
+  hidden: {
+    display: 'none'
+  }
+};
+
+var Visualization = React.createClass({
+  render: function () {
+    var appWidth = applabConstants.APP_WIDTH;
+    var appHeight = applabConstants.APP_HEIGHT - applabConstants.FOOTER_HEIGHT;
+    return (
+      <div>
+        <div id="visualization">
+          <div id="divApplab" className="appModern" tabIndex="1"/>
+          <div id="designModeViz" className="appModern" style={styles.hidden}/>
+          <svg version="1.1"
+               baseProfile="full"
+               xmlns="http://www.w3.org/2000/svg"
+               id="visualizationOverlay"
+               width={appWidth}
+               height={appHeight}
+               viewBox={"0 0 " + appWidth + " " + appHeight}
+               pointerEvents="none"/>
```

Comment:
Only IE11. Guess I should test 9/10 too, since we technically haven't cut them yet :(